Scheduler/observing_blocks_maintel/BLOCK-T299.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "BLOCK-T299",
+    "program": "BLOCK-T299",
+    "constraints": [],
+    "scripts": [
+          {
+            "name": "maintel/point_azel.py",
+            "standard": true,
+            "parameters": {
+                "az": 295.713,
+                "el": 29.095,
+                "target_name": "EUTELSAT_117W_A",
+                "reason": "SITCOM-1284",
+                "wait_dome": true
+            }
+        },
+        {
+            "name": "run_command.py",
```

Comment:
Instead of a point_azel and then a track_target script, you could combine these by using the track_target.py script with the track_azel option https://github.com/lsst-ts/ts_standardscripts/blob/ddf41fa419080b94834fb47dc399a9593e119fed/python/lsst/ts/standardscripts/base_track_target.py#L120

Reply:
Modified to use the track_target script.